src/shared/components/barGraph/BarGraph.tsx

```diff
@@ -129,74 +137,84 @@ export default class BarGraph extends React.Component<IBarGraphProps, {colors: s
             .sort((a, b) => b.caseCount - a.caseCount)
             .slice(0, 20);
 
-        Highcharts.chart(this.chartTarget, {
-
-            chart: {type: 'bar', backgroundColor:'' },
-
-            exporting: { enabled:false },
-
-            title: {text: ''},
-
-            xAxis: {categories: cancerTypeStudiesArray.slice(0, 20).map((cancer) => (cancer.shortName)),
-                    labels: {style: {fontSize: '10px'}}},
-
-            yAxis: {
-                allowDecimals: false,
-                min: 0,
-                // max: 6000,
-                title: {text: ''}
-            },
+        const datasets = _.flattenDeep(cancerTypeStudiesArray.map((cancerStudySet, i) => (
+            cancerStudySet.studies.sort((a: ICounterObj, b:ICounterObj) => +b.allSampleCount - +a.allSampleCount).map((cancerStudy: CancerStudy, j: number) => {
```

Comment:
Without the "+", I receive a TS error:
```
TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number', or an enum type.
```
I do not know why I receive this error because on line 91, it allows `allSampleCount` to be added to another number.

ICounterObj is defined on line 28.